types/actions.d.ts

```diff
@@ -0,0 +1,4 @@
+type link = (node: Element) => { destroy(): void }
+type links = (node: Element) => { destroy(): void }
```

Comment:
It should be a value, not a type. this code causing error

```
Error: 'link' only refers to a type, but is being used as a value here. (ts)
<div>
  <a href="/" use:link>home</a>
  <a href="login" use:link>login</a>
```


```suggestion
const link: (node: Element) => { destroy(): void };
const links: (node: Element) => { destroy(): void };
```

Reply:
The suggested change above also returns an error, more specifically, the following:

> ```Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier. ts(1046)```

Reply:
I tested these types in this [gist], which started by `declare`. Can you test `export`?
```
export const link: (node: Element) => { destroy(): void }
```

[gist]: https://gist.github.com/smmoosavi/4c56234c1c04d6c5e8b502ea5ab5b8f8

Reply:
Indeed, it works, i should have tested it before, taking into consideration that the error message is clear on it's suggestions, my bad